src/dispatch/incident/flows.py

```diff
@@ -630,140 +631,127 @@ def incident_create_flow(*, incident_id: int, checkpoint: str = None, db_session
-
-    # we remind the incident commander to write a tactical report
-    send_incident_report_reminder(incident, ReportTypes.tactical_report, db_session)
-
-    # we update the status of the external ticket
-    update_external_incident_ticket(incident, db_session)
+    # we un-archive the conversation
+    convo_plugin = plugin_service.get_active(db_session=db_session, plugin_type="conversation")
+    if convo_plugin:
+        convo_plugin.instance.unarchive(incident.conversation.channel_id)
 
 
-@background_task
-def incident_stable_flow(incident_id: int, command: Optional[dict] = None, db_session=None):
+def incident_stable_status_flow(incident: Incident, db_session=None):
     """Runs the incident stable flow."""
-    # we load the incident instance
-    incident = incident_service.get(db_session=db_session, incident_id=incident_id)
-
     # we set the stable time
     incident.stable_at = datetime.utcnow()
 
-    # we remind the incident commander to write a tactical report
-    send_incident_report_reminder(incident, ReportTypes.tactical_report, db_session)
+    # set time immediately
+    db_session.add(incident)
+    db_session.commit()
 
-    # we update the external ticket
-    update_external_incident_ticket(incident, db_session)
+    if incident.incident_review_document:
+        log.debug("Incident review document already created... skipping creation.")
+        return
 
-    if not incident.incident_review_document:
-        storage_plugin = plugin_service.get_active(db_session=db_session, plugin_type="storage")
-        if not storage_plugin:
-            log.warning("Incident review document not created, no storage plugin enabled.")
-            db_session.add(incident)
-            db_session.commit()
-            return
+    storage_plugin = plugin_service.get_active(db_session=db_session, plugin_type="storage")
+    if not storage_plugin:
+        log.warning("Incident review document not created, no storage plugin enabled.")
+        return
 
-        # we create a copy of the incident review document template and we move it to the incident storage
-        incident_review_document_name = f"{incident.name} - Post Incident Review Document"
-        template = document_service.get_incident_review_template(db_session=db_session)
+    # we create a copy of the incident review document template and we move it to the incident storage
+    incident_review_document_name = f"{incident.name} - Post Incident Review Document"
+    template = document_service.get_incident_review_template(db_session=db_session)
 
-        # incident review document is optional
-        if template:
-            incident_review_document = storage_plugin.instance.copy_file(
-                folder_id=incident.storage.resource_id,
-                file_id=template.resource_id,
-                name=incident_review_document_name,
-            )
+    # incident review document is optional
+    if template:
```

Comment:
We may want to check if the template exists, and if not, let the user know using a warning.